### ISSUE TITLE
Cljs (exists? foo.bar/az) complains about require

### ISSUE THREAD

**Issue by Cyrik** [clj-kondo/clj-kondo#1472]
**version**

v2021.10.19

**platform**

LSP, but happens on cli version as well.

**editor**

Calva

**problem**

clj-kondo complains that a namespace doesn't exist, even when checking for it:

**repro**

```
$ clj-kondo --lint - --lang cljs <<< '(exists? foo.bar/az)'
<stdin>:1:10: warning: Unresolved namespace foo.bar. Are you missing a require?
```

**expected behavior**

Should not complain at this point.
